ivy/functional/ivy/gradients.py

```diff
@@ -268,7 +269,21 @@ def _is_variable(x, exclusive=False) -> bool:
     )
 
 
-def _variable_data(x):
+@outputs_to_ivy_arrays
```

Comment:
This decorator is not needed as this function is to be used inside other functions

Reply:
I have removed the decorator in the latest changes. Thank you!